CONTRIBUTING.md

```diff
@@ -350,7 +350,23 @@ by running the test suite in a Docker environment that simulates Swift on Linux.
     cd swift-docc
     swift run docc
     ```
-    
+
+## Updating Build Rules
+
+In order to build DocC as part of the toolchain distribution, a parallel CMake
+based build exists. Note that this is **not** supported for development purposes
+(you cannot execute the test suite with this build).
+
+CMake requires that the full file list is kept up-to-date. When adding or
+removing files in a given module, the `CMakeLists.txt` list must be updated to
+the file list.
+
```

Comment:
Is there anything the developer needs to do if they add a new target or a new dependency?

Reply:
Yes, a new target would require a new target to be defined (https://cmake.org/cmake/help/latest/command/add_library.html)